src/precomputed_quantities.jl

```diff
@@ -16,10 +16,14 @@ end
 function precomputed_quantities!(Y, p, t, colidx)
     ᶜuₕ = Y.c.uₕ
     ᶠw = Y.f.w
-    (; ᶜuvw, ᶜK, ᶜts, ᶜp, params) = p
-    (; ᶜinterp) = p.operators
+    (; ᶠuvw, ᶜuvw, ᶜK, ᶜts, ᶜp, params) = p
```

Comment:
Is storing `ᶠuvw` the best way of doing this? (@simonbyrne)

Reply:
For now, I think so, as we need it in a few places. If it turns out we don't, we can remove it later

Reply:
We can probably remove some other variables (e.g. `ᶠu³`)

Reply:
Leaving `ᶠu³` in place for now since it is used in several places within the `implicit_tendency` computations.